profile/README.adoc

```diff
@@ -40,18 +40,21 @@ A communication protocol that enables developers to build apps and services that
 * Same subscription API across all platforms, formalized as a service itself (uSubscription service) where the subscription flows across platforms and across different transports automatically
   ** Implementation may be custom to a given platform (not relying on generated code)
 
-=== uProtocol Client Library (uPClient)
-uPClient library is the *only* client-facing library that application and service developers require to connect & communication to an <<uProtocol Implementation>>. 
+== How to Get Started? 
 
-uPClient libraries are easy to locate, they have the name `up-client-<IMPLEMENTATION>-<LANGUAGE>`, example https://github.com/eclipse-uprotocol/up-client-android-java[up-client-android-java], https://github.com/eclipse-uprotocol/up-client-zenoh-rust[up-client-zenoh-rust], etc...
+1. *Pick a Programming Language:* To build applications and services (uEs) that uses uProtocol, you'll first need to pick a programming languages, such as: https://github.com/eclipse-uprotocol/up-rust[Rust], https://github.com/eclipse-uprotocol/up-java[Java], https://github.com/eclipse-uprotocol/up-cpp[C++], https://github.com/eclipse-uprotocol/up-python[Python], and https://github.com/eclipse-uprotocol/up-kotlin[Kotlin]. Each library contains a `README` that will explain how to build and use uProtocol for their programming language.
 
-If you notice a language or implementation missing when you search below, and you would like to add one, check out <<How to Get Involved>> section below.
+2. *Pick a Transport:* Next you'll need to decide the communication middleware (transports) your uE will run on top of, some options are Eclipse Zenoh, SOME/IP, MQTT 5, Android (Binder), etc...  Check out the list of projects below to find a combination of language and transport that suits your needs, if you don't find one, we would love a contribution back :-).
```

Comment:
*Select a Transport:* Next choose the communication middleware (transports) your uE will connect to, some options are...